.github/workflows/update-interfaces.yml

```diff
@@ -0,0 +1,55 @@
+name: Update protobuf stubs
+
+on:
+  push:
```

Comment:
Probably only want this running on main (once it is known to work), and possibly with a path filter to just when `api-interfaces` is changed.